### PR TITLE
feat: Move default git provider integration settings page DOCS-598

### PR DESCRIPTION
**Pull request by nicklem**
Moves the page "Configuring default Git provider integration settings" from `organizations` to `organizations/integrations`

### :eyes: Live preview
[Default Git provider integration settings](https://docs-598-move-org-level-git-provide--docs-codacy.netlify.app/organizations/integrations/default-git-provider-integration-settings/)

### :construction: To do
- [ ] After merging, also merge https://github.com/codacy/codacy-spa/pull/1459

### CODE REVIEW

File: mkdocs.yml [hunk 1]
```diff
@@ -599,8 +600,8 @@ nav:
           - organizations/using-gate-policies.md
           - organizations/using-coding-standards.md
           - organizations/copying-code-patterns-between-repositories.md
-          - organizations/configuring-default-git-provider-integration-settings.md
           - Integrations:
```
Comment (claudiacarpinteiro) [inline mkdocs.yml:603]: What if we take the chance to use a gerund here, for information architecture consistency? 🤔 

```suggestion
          - Managing integrations:
```

**Review by claudiacarpinteiro — COMMENTED** [review]
Nice update, @nicklem! 👌 
I left one comment for your evaluation.

File: mkdocs.yml (hunk 1 above)
Comment (nicklem) [reply mkdocs.yml]: Good idea. I also renamed the one under the "Configuring your repositories" section.

**Review by claudiacarpinteiro — APPROVED** [review]
LGTM! 🚀

**Pull request merged**